www/src/assets/icons/layer-icon.js

```diff
@@ -0,0 +1,324 @@
+          cy="16.7765"
+          r="1.01819"
+          transform="rotate(90 2.44852 16.7765)"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+        />
+        <circle
+          cx="6.62354"
+          cy="6.69713"
+          r="1.01819"
+          transform="rotate(45 6.62354 6.69713)"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+        />
+        <circle
+          cx="6.62354"
+          cy="26.8563"
+          r="1.01819"
+          transform="rotate(45 6.62354 26.8563)"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+        />
+        <path
+          d="M16.7031 2.52197C24.5758 2.52197 30.9578 8.90399 30.9578 16.7766C30.9578 24.6492 24.5758 31.0313 16.7031 31.0313C12.9071 31.0313 9.45762 29.5474 6.90308 27.1282"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+        />
+      </svg>
+    ),
+    Build: (
+      <svg
+        width="37"
+        height="37"
+        viewBox="0 0 37 37"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M30.2955 12.3557C27.4818 16.5214 24.2529 20.3909 20.6584 23.905C12.243 32.3204 3.8995 37.6216 2.02247 35.7399C0.686096 34.4036 3.04575 29.7377 7.52069 24.157"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+        <path
+          d="M20.5393 11.0486C22.1007 9.75895 23.7323 8.55666 25.4266 7.44727"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+        <path
+          d="M16.1203 11.2441C10.4419 6.66224 5.66914 4.21859 4.31749 5.56565C-0.607995 10.4911 29.4428 39.647 33.3405 35.7402C34.4981 34.581 32.922 30.9567 29.5635 26.3718"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+        <path
+          d="M12.0278 13.1609C14.1006 11.6304 16.629 10.8434 19.2043 10.9272C21.7796 11.011 24.2514 11.9606 26.2204 13.6226C28.1894 15.2847 29.5405 17.562 30.0554 20.0867C30.5703 22.6114 30.219 25.236 29.0581 27.5364C27.8972 29.8367 25.9947 31.6785 23.6579 32.7641C21.3211 33.8497 18.6865 34.1157 16.1798 33.5192C13.6732 32.9226 11.4408 31.4983 9.84357 29.4765C8.2463 27.4546 7.37737 24.9532 7.3772 22.3766C7.37737 21.9105 7.40542 21.4449 7.4612 20.9821"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+        <path
+          d="M32.5543 8.65346C34.4648 8.65346 36.0136 7.10468 36.0136 5.19416C36.0136 3.28364 34.4648 1.73486 32.5543 1.73486C30.6438 1.73486 29.095 3.28364 29.095 5.19416C29.095 7.10468 30.6438 8.65346 32.5543 8.65346Z"
+          stroke={fillColor}
+          strokeWidth="1.52728"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        />
+      </svg>
+    ),
+    Data: (
```

Comment:
Little nit-picky, but we can clean this file up quite a bit if we break these SVG icons out into either their raw SVG equivalent (and import with raw-loader) or as a React component and then the file would look like:

```js
import React from 'react'

import Data from './data'

const lookup = {
  Data
}

const LayerIcon = ({ name }) => {
  const Icon = lookup[name]
  return Icon ? <Icon {...whateverProps /> : null
}
```

Reply:
However -- since these seem to only be used by _this file_ it could be prematurely optimizing a bit :) Up to you!

Reply:
Yeah I think I'm going to go the path of least resistance and just keep them in the same file until it feels more necessary to break them up 😬